client/src/panels/people/employee_type.js

```diff
@@ -22,6 +24,12 @@ const { tenure } = businessConstants;
 const { A11YTable } = declarative_charts;
 
 
+const employee_type_glossary_footnotes = _.map(
+  ['INDET_PEOPLE', 'TERM_PEOPLE', 'CASUAL_PEOPLE', 'STUD_PEOPLE'],
+  (glossary_key) => ({text: GlossaryEntry.lookup(glossary_key).definition})
+);
```

Comment:
Just occurred to me that this kind of hack will interfere with (or at least be a special case in) #44, if it goes in the direction of being more than just an RPB feature.

Reply:
Can we generate footnotes from certain glossary items in `populate_footnotes.js`? That avoids the multiple sources of truth problem, and lets us display them like footnotes if we want.

Reply:
Could do. I was also thinking that it might be good to tag panels with relevant glossary entries, make it standard. There's lots of panels that might have relevant definitions that don't show up exactly in text and, as a result, don't get direct glossary tooltips/links. Maybe put them in a details drawer like the footnotes (or rename the existing one to Footnotes & Definitions), or add a little dictionary icon to the utils bar to bring up a definitions modal.

Reply:
I guess I should capture that thought in an issue, not doing it in this PR.